packages/bsky/src/index.ts

```diff
@@ -116,6 +117,12 @@ export class BskyAppView {
       adminPasses: config.adminPasswords,
     })
 
+    if (config.statsigKey) {
+      Statsig.initialize(config.statsigKey, {
+        environment: { tier: config.statsigEnv },
+      }).catch((_) => {})
```

Comment:
My understanding of the statsig sdk is that it needs to make it into a "ready" state before you can start checking gates, and awaiting `initialize()` is how you ensure you're in that state.  By silently ignoring the error here, we could end-up not in a ready state and then I'm pretty sure checking gates would always fail.

We should be able to move the await into the context's lifecycle `start()`.  By a similar token we also want to shut statsig down cleanly so that the process doesn't hang on shutdown due to open handles.  And we can perform that in the context's lifecycle `destroy()`.

Reply:
My understanding based on the docs is that even if this fails to load the fresh gates, setup has already occurred and gate checks will return `false` by default, not throw.

But ah yes, good call, I didn't realize how the lifecycle worked here! I'll move that in. Actually started this branch by adding to app context but reverted for the simpler option 🙃